mkosi.md

```diff
@@ -237,6 +237,16 @@ details see the table below.
   run, when specified once the incremental cache files are deleted
   too, and when specified twice the package cache is also removed.
 
+`--gpt-first-lba`
+
+: Override the first usable LBA (Logical Block Address) within the
+  GPT header. This defaults to `2048` which is actually the desired value.
+  However, some tools, e.g. the `prl_disk_tool` utility from the
+  Parallels virtualization suite require this to be set to `34`, otherwise
+  they might fail to resize the disk image and/or partitions inside it.
+  Should you have the need the override this, you can do this here and
+  go as low as `34`. (Optional)
```

Comment:
Just a nitpick, but I think this is more in line with the tone of the rest of the docs.
```suggestion
: Override the first usable LBA (Logical Block Address) within the
  GPT header. This defaults to `2048` which is the desired value,
  however, some tools, e.g. the `prl_disk_tool` utility from the
  Parallels virtualization suite require this to be set to `34`, otherwise
  they might fail to resize the disk image and/or partitions inside it.
```
The last two lines are not needed after the excellent example.